src/_common/content/content-editor/plugins/update-incrementer-plugin.ts

```diff
@@ -22,6 +22,11 @@ export class UpdateIncrementerPlugin {
 		}
 		// If anything in the editor changes (content or selection), make sure we increment, so we can for example reposition controls.
 		this.appEditor.stateCounter++;
+
+		if (this.appEditor.canShowMentionSuggestions > 0) {
```

Comment:
Explain, or at least point to detectMentionSuggestions input rule for more info.